var/spack/repos/builtin/packages/libunistring/package.py

```diff
@@ -13,5 +13,6 @@ class Libunistring(AutotoolsPackage):
     homepage = "https://www.gnu.org/software/libunistring/"
     url      = "https://ftpmirror.gnu.org/libunistring/libunistring-0.9.7.tar.xz"
 
+    version('0.9.10', sha256='eb8fb2c3e4b6e2d336608377050892b54c3c983b646c561836550863003c05d7')
```

Comment:
Apologies but when I merged the newer https://github.com/spack/spack/pull/11255 it created a conflict here. You can drop this edit as that PR adds the same version